core/sdk/src/api/config.rs

```diff
@@ -32,7 +32,7 @@ use super::manifest::{
 
 use super::manifest::device_manifest::AppLibraryEntry;
 
-pub const FEATURE_CLOUD_PERMISSIONS: &str = "cloud_permissions";
+pub const FEATURE_DISTRIBUTOR_SESSION: &str = "distributor_session";
```

Comment:
Just making sure you meant this to be part of your commit?

Reply:
Repurposed for authentication.session call